### PR TITLE
Station header improvements

### PR DESCRIPTION
**Pull request by evulins**
Fixed small UI bugs and responsiveness in Station header 
[Trello card](https://trello.com/c/DJznR4kx/1789-station-header-improvements)

### CODE REVIEW

File: app/javascript/react/components/molecules/FixedStreamStationHeader/atoms/StationActionButtons/StationActionButtons.tsx [hunk 1]
```diff
@@ -35,12 +29,15 @@ const StationActionButtons = () => {
...
+        {
+          // In MVP we don't have this feature
+          /* <Button
           onClick={() => {}}
           aria-labelledby={t("calendarHeader.altExportSession")}
         >
           {t("calendarHeader.exportSession")} <img src={downloadImage} />
-        </Button>
+        </Button> */
+        }
```
Comment (franiakat) [inline app/javascript/react/components/molecules/FixedStreamStationHeader/atoms/StationActionButtons/StationActionButtons.tsx:40]: I'd normally be removing such pieces, my take is that we hope to be doing the post MVP work, but I am not 100% sure. :)

File: app/javascript/react/store/fixedStreamSelectors.ts [hunk 2]
```diff
@@ -137,11 +137,14 @@ const selectFixedStreamShortInfo = createSelector(
   (fixedStreamData, lastDailyAverage): FixedStreamShortInfo => {
     const { value: lastMeasurementValue, date } = lastDailyAverage || {};
     const lastMeasurementDateLabel = moment(date).format("MMM D");
+    const formattedDate = moment(fixedStreamData.stream.lastUpdate).local().format("HH:mm, MMM D YYYY");
+    const lastUpdate = formattedDate + " (local time)";
```
Comment (franiakat) [inline app/javascript/react/store/fixedStreamSelectors.ts:141]: I'd return only the formatted date and cover the local time addition with translations, there is a way to pass a variable content with translation key, where you can define the final structure of your label. 
We have such cases in the app already, lmk if it's not clear.

File: db/schema.rb [hunk 3]
```diff
@@ -78,7 +78,7 @@
     t.integer "stream_id"
     t.integer "milliseconds", default: 0
     t.float "measured_value"
-    t.geometry "location", limit: {:srid=>4326, :type=>"geometry"}, null: false
+    t.geometry "location", limit: {:srid=>0, :type=>"geometry"}, null: false
```
Comment (franiakat) [inline db/schema.rb:81]: Hm, I'm wondering if this shouldn't be here, I'd say it shouldn't :D cc. @BartkiewiczM

File: app/javascript/react/components/molecules/FixedStreamStationHeader/atoms/ValueLabel/ValueLabel.style.tsx [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import styled from "styled-components";
-import { blue, mint } from "../../../../../assets/styles/colors";
+import { blue, green, mint } from "../../../../../assets/styles/colors";
```
Comment (franiakat) [inline app/javascript/react/components/molecules/FixedStreamStationHeader/atoms/ValueLabel/ValueLabel.style.tsx:2]: I see mint is unused, not connected to your changes, though.

File: app/javascript/react/components/molecules/FixedStreamStationHeader/FixedStreamStationHeader.style.tsx [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import styled from "styled-components";
 
-import { H4 } from "../../Typography";
+import { H4, H5 } from "../../Typography";
```
Comment (franiakat) [inline app/javascript/react/components/molecules/FixedStreamStationHeader/FixedStreamStationHeader.style.tsx:3]: Unused H4.

**Review by franiakat — CHANGES_REQUESTED** [review]
I am wondering, given we're left with only action button in header, how about keeping normal label in the button?

File: app/javascript/react/components/molecules/FixedStreamStationHeader/atoms/StationActionButtons/StationActionButtons.tsx (hunk 1 above)
Comment (evulins) [reply app/javascript/react/components/molecules/FixedStreamStationHeader/atoms/StationActionButtons/StationActionButtons.tsx:40]: Done

File: app/javascript/react/store/fixedStreamSelectors.ts (hunk 2 above)
Comment (evulins) [reply app/javascript/react/store/fixedStreamSelectors.ts:141]: Done

File: app/javascript/react/components/molecules/FixedStreamStationHeader/atoms/ValueLabel/ValueLabel.style.tsx (hunk 4 above)
Comment (evulins) [reply app/javascript/react/components/molecules/FixedStreamStationHeader/atoms/ValueLabel/ValueLabel.style.tsx:2]: Done

File: app/javascript/react/components/molecules/FixedStreamStationHeader/FixedStreamStationHeader.style.tsx (hunk 5 above)
Comment (evulins) [reply app/javascript/react/components/molecules/FixedStreamStationHeader/FixedStreamStationHeader.style.tsx:3]: Done

File: db/schema.rb [hunk 6]
```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2024_03_13_144943) do
+ActiveRecord::Schema.define(version: 2024_03_22_101215) do
```
Comment (franiakat) [inline db/schema.rb:13]: @evulins I think this should stay the same. 🙃

**Review by franiakat — APPROVED** [review]
🎖️ LGTM!

**Pull request merged**